content/en/tracing/trace_collection/custom_instrumentation/python/ddtrace-api.md

```diff
@@ -0,0 +1,256 @@
+---
+title: Python Custom Instrumentation using ddtrace-api-py (pre-release)
+description: '(pre-release) Manually instrument your Python application to send custom traces to Datadog.'
+code_lang: dd-api-alpha
+type: multi-code-lang
+code_lang_weight: 1
+further_reading:
+    - link: 'tracing/other_telemetry/connect_logs_and_traces'
+      tag: 'Documentation'
+      text: 'Connect your Logs and Traces together'
+    - link: 'tracing/glossary/'
+      tag: 'Documentation'
+      text: 'Explore your services, resources, and traces'
+---
+
+*Note: the ddtrace-api Python package is in a pre-release state and may contain significant issues. If you'd rather use something more stable, use the [ddtrace API][9].*
```

Comment:
Instead of `ddtrace API`, I think it's better to call this `Datadog API` to be consistent with the navigation bar.

Reply:
Maybe, but it ought to be clear, maybe in parentheses, that this refers to the original `ddtrace` package.